frontend/src/ducks/feeStats.ts

```diff
@@ -0,0 +1,120 @@
+import StellarSdk from "stellar-sdk";
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import { RootState } from "config/store";
+import { getErrorString } from "helpers/getErrorString";
+import {
+  ActionStatus,
+  AverageTransactionFeeData,
+  FeeStatsData,
+  FeeStatsInitialState,
+  Network,
+  RejectMessage,
+} from "types";
+import { networkConfig } from "constants/settings";
+import BigNumber from "bignumber.js";
+
+export const fetchFeeStatsDataAction = createAsyncThunk<
+  FeeStatsData,
+  void,
+  { rejectValue: RejectMessage; state: RootState }
+>("feeStats/fetchFeeStatsDataAction", async (_, { rejectWithValue }) => {
+  try {
+    const server = new StellarSdk.Server(networkConfig[Network.MAINNET].url);
+
+    const response = server.feeStats();
```

Comment:
idk why typescript allows this but there should be an await here

```suggestion
    const response = await server.feeStats();
```